RFS/src/main/java/org/opensearch/migrations/bulkload/transformers/TransformerToIJsonTransformerAdapter.java

```diff
@@ -24,7 +24,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-
+/**
+ * Adapts an {@link IJsonTransformer} to a {@link Transformer}.
+ */
 @Slf4j
 public class TransformerToIJsonTransformerAdapter implements Transformer {
```

Comment:
Is the name of this class backward?  It feels like the IJsonTransformerAdapter is being transformed into a Metadata Transformer interface, not the other way around.